test/mock/lib/request.js

```diff
@@ -1,4 +1,9 @@
-var Long = require('bson').Long;
+var Long = require('bson').Long,
+    Snappy = require('snappy');
+    zlib = require('zlib');
+
+var SNAPPY_ID = 1,
```

Comment:
ah, looks like we have constants here. When you use them in the module itself, you can drop these and share the code